src/pages/DispensingPage.js

```diff
@@ -349,10 +350,16 @@ const mapDispatchToProps = dispatch => ({
 
   editPatient: patient =>
     batch(() => {
-      dispatch(NameNoteActions.createSurveyNameNote(patient));
+      dispatch(NameNoteActions.createSurveyNameNote(UIDatabase.get('Name', patient)));
```

Comment:
`patient` was actually `patientID`- and `createSurveyNameNote` was changed to accept a patient, not the ID